jobrunner/cli/flags.py

```diff
@@ -0,0 +1,88 @@
+"""
+Ops utility getting and setting db flags
+"""
+import argparse
+import sqlite3
+import sys
+
+from jobrunner.lib.database import create_table, get_connection, select_values
+from jobrunner.models import Flag
+from jobrunner.queries import get_flag, set_flag
+
+
+def parse_cli_flag(raw):
+    """Do you have a flaaaaaag?"""
+    name, equals, value = raw.partition("=")
+    if not equals:
+        raise argparse.ArgumentError(f"set must have for {name}=value, not just {name}")
+    if value == "":
+        value = None
+    return name, value
+
+
+def set_flags(flags, create):
+    try:
+        for name, value in flags:
+            set_flag(name, value)
+
+    except sqlite3.OperationalError as e:
+        if "no such table" in str(e):
+            if create:
+                create_table(get_connection(), Flag)
+                # try again
+                for name, value in flags:
+                    set_flag(name, value)
+            else:
+                sys.exit(
+                    "Flags table does not exists. Run command again with --create to create it."
+                )
+        else:
+            raise
+
+    return [p[0] for p in flags]
+
+
+def main(action, flags, create=False):
+    if action == "set":
+        flags = set_flags(flags, create)
+    elif not flags:
+        try:
+            flags = select_values(Flag, "id")
+        except sqlite3.OperationalError:
+            pass
```

Comment:
`flags` won't be set so you'll get an error further down when you try to iterate it. Is is worth printing something like "flags database table does not exist" and exiting? Or could just set `flags = []` as the simplest fix.

Reply:
So, currently, when there's no table, `get` "just works", and `set` fails.

I think that's a good mode for the internal API to use.

However, with the flags cli command, there's always a user running it, so perhaps we should be strict about the table's existence for both `get` and `set`.

So we could support --create logic for both?

I think that's probably the best approach, I'll start on that, yell if you think otherwise.